recipes-extended/wxwidgets/wxwidgets_3.0.3.1.bb

```diff
@@ -4,45 +4,37 @@ HOMEPAGE = "http://www.wxwidgets.org"
 LICENSE = "WXwindows"
 LIC_FILES_CHKSUM = "file://docs/licence.txt;md5=18346072db6eb834b6edbd2cdc4f109b"
 
-DEPENDS = "webkit-gtk gstreamer gtk+ jpeg tiff libpng zlib expat libxinerama libglu"
+DEPENDS = "webkitgtk gstreamer gtk+ jpeg tiff libpng zlib expat libxinerama libglu"
 
-SRC_URI = "${SOURCEFORGE_MIRROR}/wxwindows/wxWidgets-${PV}.tar.bz2"
-SRC_URI[md5sum] = "e98c5f92805493f150656403ffef3bb0"
-SRC_URI[sha256sum] = "b74ba96ca537cc5d049d21ec9ab5eb2670406a4aa9f1ea4845ea84a9955a6e02"
+SRC_URI = "https://github.com/wxWidgets/wxWidgets/releases/download/v${PV}/wxWidgets-${PV}.tar.bz2"
+SRC_URI[md5sum] = "694ed5f5ea1597e06a9adc6f347d8929"
+SRC_URI[sha256sum] = "3164ad6bc5f61c48d2185b39065ddbe44283eb834a5f62beb13f1d0923e366e4"
 
 S = "${WORKDIR}/wxWidgets-${PV}"
 
-inherit autotools-brokensep pkgconfig binconfig
-
-EXTRA_AUTORECONF = " -I ${S}/build/aclocal"
-EXTRA_OECONF = "  --with-opengl \
-                 --without-sdl \
-                 --enable-mediactrl=yes \
-                 --enable-webviewwebkit=yes \
-                 --disable-gpe \
-                 --disable-visibility \
-                 --disable-rpath \
-               "
+#inherit autotools pkgconfig binconfig
+#inherit autotools
+inherit autotools pkgconfig
 
+EXTRA_OECONF = "  --disable-shared \
+	--enable-unicode \
+	"
 CXXFLAGS := "${@oe_filter_out('-fvisibility-inlines-hidden', '${CXXFLAGS}', d)}"
 CXXFLAGS += "-std=gnu++11"
 
-# Broken autotools :/
 do_configure() {
 	oe_runconf
 }
 
 # wx-config contains entries like this:
 # this_prefix=`check_dirname "/build/v2013.06/build/tmp-angstrom_v2013_06-eglibc/work/cortexa8hf-vfp-neon-angstrom-linux-gnueabi/wxwidgets/2.9.5-r0/wxWidgets-2.9.5"`
 do_install_prepend() {
```

Comment:
An empty `do_install_prepend` and its comment can just be dropped if it is not needed anymore.